Pipfile

```diff
@@ -13,3 +13,7 @@ pytest = ">=2.7.2"
 
 [requires]
 python_version = "3.10"
+
+[packages.e1839a8]
```

Comment:
I am not a pip expert, but previously for VSS I got the impression that it is good practice to always have a Pipfile.lock as well as the Pipfile to "lock" the build to specific versions. Or do we see that as not needed here?

Reply:
I will bring it back later. There were issues with pipenv on Linux. During the experiments I have removed it.

Reply:
@erikbosch I have reverted `Pipenv.lock` file back, please check and resolve the thread.